fhir-path/src/main/java/com/ibm/fhir/path/function/ToStringFunction.java

```diff
@@ -38,9 +38,7 @@ public int getMaxArity() {
 
     @Override
     public Collection<FHIRPathNode> apply(EvaluationContext evaluationContext, Collection<FHIRPathNode> context, List<Collection<FHIRPathNode>> arguments) {
-        if (!isSingleton(context)) {
-            return empty();
-        }
+        checkStringValue(context);
```

Comment:
toString doesn't exist under http://hl7.org/fhirpath/N1/#string-manipulation and so shouldn't use this check.  

instead this one falls under section 5.5 which says this:
> The functions in this section operate on collections with a single item. If there is more than one item, the evaluation of the expression will end and signal an error to the calling environment.